### PR TITLE
Deploy manifest and sqd init

### PR DESCRIPTION
**Pull request by dzhelezov**

### CODE REVIEW

File: docs/deploy-squid/migration.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Migrate from older versions of `@subsquid/cli`.
+
+Since `@subsquid/cli@1.x` the commands `squid:deploy` and `squid:update` are deprecated in favor of the manifest-based deployments and `sqd deploy`.
+
+To migrate:
+
+- Create `squid.yaml` in the squid root folder
+- Copy the deployment manifest from e.g. [squid-evm-template](https://github.com/subsquid/squid-evm-template/blob/main/squid.yaml)
+- Update the squid name, description and version
```
Comment (RaekwonIII) [inline docs/deploy-squid/migration.md:14]: I just wanted to suggest to clarify that this point and the next 2 are to be performed on the `squid.yaml` file.
Something like:
 - copy the content of `squid.yaml` file from squid-evm-template
 - edit the local file, updating the squid name, description, and version
 - update (if necessary) ...

Comment (belopash) [reply docs/deploy-squid/migration.md:14]: maybe not just "copy file from", but also add a link to manifest docs?

Comment (belopash) [reply docs/deploy-squid/migration.md:14]: and why from `squid-evm-template`?

Comment (belopash) [reply docs/deploy-squid/migration.md:14]: i mean better to provide better to provide content of manifest right there: "copy the following lines"

**Pull request merged**